guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,159 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+LEAPP is the supported tool for {RHEL} 7 to {RHEL} 8 upgrades.
```

Comment:
Here you say upgrading from RHEL7 to RHEL8, later you write it needs to run on CentOS 7 and upgrades on RHEL are not possible. This will confuse all readers :)

Just to make it clear:
* Foreman and Katello installations running on CentOS 7 can be upgraded to CentOS Stream 8 (and probably any other EL8-rebuild like Rocky and Alma, even tho I didn't test that explicitly).
* Foreman and Katello installations running on RHEL7 can't be upgraded (right now).
* Satellite installations will be upgradeable from RHEL7 to RHEL8, no CentOS/EL-rebuilds.